### PR TITLE
intent doc fixed 

### PR DESCRIPTION
**Pull request by smartyrad**
Addresses #4

### CODE REVIEW

**Comment by adammichaelwood** [comment]
This is coming along. 

Can you please add a paragraph at the top of the document explaining what an intent is in the context of Collect, and what they are used for?

**Comment by smartyrad** [comment]
@adammichaelwood Sure :+1:

File: intents.rst [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+This will allow user to choose the form and go on filling it .
+
+Similarly for an instance of the form : 
+
+.. code-block:: java
+
+	Intent intent = new Intent(android.intent.action.EDIT);
+	intent.setData(org.odk.collect.android.provider.InstanceProviderAPI.InstanceColumns.CONTENT_URI);
+
+Apart from this , we also need to put in a mode, which will allow to differentiate between editing or viewing form . So we need to put in an extra here , which goes like :
```
Comment (lognaturel) [inline intents.rst]: This is no longer right. We should not document extras, the code above should only be for launching form edits.

File: intents.rst [hunk 2]
```diff
@@ -0,0 +1,136 @@
+Collects's Intent
```
Comment (lognaturel) [inline intents.rst]: I think this should be something like Launching ODK Collect From External Apps.

File: intents.rst [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+==================
+
+About Intent
```
Comment (lognaturel) [inline intents.rst]: I think this section should focus on why someone might want to launch Collect from their app. Linking out to the Android docs is enough general background on Intents.

**Comment by lognaturel** [comment]
Thanks for looking into this @smartyrad and working with us to figure out what the right content is!

**Comment by adammichaelwood** [comment]
@smartyrad 
Can you address @lognaturel's reviews and update the PR?
Thanks.

**Comment by adammichaelwood** [comment]
@smartyrad Are you still working on this?

**Comment by adammichaelwood** [comment]
Closing due to inactivity.

**Pull request closed without merging**